internal/pkg/service/common/dependencies/mocked.go

```diff
@@ -242,6 +242,11 @@ func newMockedConfig(t *testing.T, opts []MockedOption) *MockedConfig {
 		opt(cfg)
 	}
 
+	if cfg.debugLogger == nil {
+		cfg.debugLogger = log.NewDebugLogger()
+		cfg.debugLogger.ConnectTo(testhelper.VerboseStdout())
+	}
+
```

Comment:
Debug logger v testoch, ktore pouzivaju `dependencies.Mocked` sa by default napoji na `stdout`, ... ked je zapnuta ENV `TEST_VERBOSE=true`. 

Zide sa to asi v kazdom teste a nemusi sa to nastavovat samostatne, vid zmazane riadky v tomto commite.